### PR TITLE
feat: new bls keys

### PR DESCRIPTION
**Pull request by antouhou**
## Issue being fixed or feature implemented

Update bls lib to match core v19

## What was done?

Migrated to new bls lib version

## How Has This Been Tested?

It hasn't

## Breaking Changes

The generated config won't work with anything less than dashd 0.19.0

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by strophy — APPROVED** [review]

File: lib/configGenerator/generateAnsibleConfig.js [hunk 1]
```diff
@@ -133,11 +133,11 @@ async function generateAnsibleConfig(network, networkName, masternodesCount, see
     const collateralPrivateKey = new PrivateKey(undefined, network);
 
     const blsSignatures = await BlsSignatures();
-    const { PrivateKey: BlsPrivateKey } = blsSignatures;
+    const { PrivateKey: BlsPrivateKey, BasicSchemeMPL } = blsSignatures;
 
     const randomBytes = new Uint8Array(crypto.randomBytes(256));
-    const operatorPrivateKey = BlsPrivateKey.fromBytes(randomBytes, true);
-    const operatorPublicKey = operatorPrivateKey.getPublicKey();
+    const operatorPrivateKey = BlsPrivateKey.from_bytes(randomBytes, true);
```
Comment (strophy) [inline lib/configGenerator/generateAnsibleConfig.js:139]: This line fails for me with output `6014136`. I think you need to do `gen_key()` with the random bytes first, then `sk.serialize()` the resulting key as input to this function? I had this error when I tried as well.

**Review by strophy — DISMISSED** [review]

**Review by PastaPastaPasta — APPROVED** [review]
specifically 77e0dcfa534776d76afc07698f9c8b1dffd3d06d LGTM

**Review by PastaPastaPasta — DISMISSED** [review]

**Review by ktechmidas — DISMISSED** [review]
LGTM

**Comment by strophy** [comment]
@antouhou let's close this in favour of #431? Relevant commit is [here](https://github.com/dashpay/dash-network-deploy/pull/431/commits/5d4c38ba380bb05a96875a48b56f43feb757a52b).

**Pull request closed without merging**